server/app/mutations/grids/update.rb

```diff
@@ -23,6 +23,14 @@ class Update < Mutations::Command
       array :default_affinity do
         string
       end
+      hash :logs do
+        required do
+          string :forwarder, matches: /^(fluentd|none)$/ # Only fluentd now supported, none removes log shipping
```

Comment:
Sadly the resulting errors are a bit obtuse either way, although with `in`, the resulting error could include the valid options...

### `matches: ...`
```
$ kontena grid update --log-forwarder asdf development
 [error] {"logs"=>{"forwarder"=>"Forwarder isn't in the right format"}}
```

### `in: ...`
```
$ kontena grid update --log-forwarder asdf development
 [error] {"logs"=>{"forwarder"=>"Forwarder isn't an option"}}
```